.github/ISSUE_TEMPLATE/simeng-bug-or-issue-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: SimEng Bug or Issue Report
+about: Report a bug or un-expected SimEng behaviour
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Check List**
+ [ ] The binary I am trying to run has been compiled statically for either RV64 or AArch64.
+ [ ] The compiled binary is a Linux Elf file.
+ [ ] I have provided both a config file and a binary to SimEng as runtime arguments.
+
+**System Description**
+ - Operating System
+ - Compiler used to compile SimEng and version
+ - Compiler used to compile the static binary and version
+ - ISA binary is targetting (i.e. armv8.4-a+sve)
```

Comment:
I wouldn't use shorthand especially considering the international audience. E.g. "Target ISA of the binary" could be clearer.

Again you could force them to find the information through a tool like the following: 

```
ra18837@IT086384:~/PhD/Programs/STREAM$ readelf -A stream.rv64g.12.2.static
Attribute Section: riscv
File Attributes
  Tag_RISCV_stack_align: 16-bytes
  Tag_RISCV_arch: "rv64i2p1_m2p0_a2p1_f2p2_d2p2_zicsr2p0_zifencei2p0_zmmul1p0"
  Tag_RISCV_priv_spec: 1
  Tag_RISCV_priv_spec_minor: 11

```

I.e. "Please provide the output of `readelf -A <binary>` (ensure capital `A` for the flag)"

Reply:
Although the readelf tool on my local machine doesn't like arm binaries as much. If run on a native machine (e.g. XCI node) this might work better but I haven't tested it

Reply:
I can change the acronym, although I'd assume someone working with SimEng would know what ISA stood for 😅

I'm also wary of asking for that tool's output if it isn't robust...

Reply:
As per offline discussion, the list item in question has been updated